docs/project/2024/mohamad_mario.sakka/index.md

```diff
@@ -0,0 +1,102 @@
+# DIY Laptop Cooling Pad
+
```

Comment:
Please add a one line description of the project.

Reply:
Done.